ansible/roles/touchforms/templates/application.properties.j2

```diff
@@ -1,3 +1,24 @@
 commcarehq.host=https://{{ SITE_HOST }}
 server.port=8181
-redis.hostname={{ groups.redis.0 }}
+user.suffix=commcarehq.org
+logging.level.org.springframework.web.filter.CommonsRequestLoggingFilter=INFO
+
+// Properties for HQ db
+datasource.hq.driverClassName=org.postgresql.Driver
+datasource.hq.url=jdbc:postgresql://{{ groups.postgresql.0 }}:6432/commcarehq
+datasource.hq.username={{ postgres_users.devreadonly.username }}
```

Comment:
for peace of mind i only gave readonly to formplayer so we don't have to worry about it accidentally deleting something in the main db